src/config/eslintrc.js

```diff
@@ -40,11 +47,32 @@ module.exports = {
     'require-yield': 2,
     'max-nested-callbacks': [2, 4],
     'max-depth': [2, 4],
-    'valid-jsdoc': [2, {
-      requireReturn: false,
-      requireParamDescription: false,
-      requireReturnDescription: false
-    }],
-    'require-await': 2
+    'require-await': 2,
+    'jsdoc/check-alignment': 2,
+    'jsdoc/check-examples': 0,
+    'jsdoc/check-indentation': 2,
+    'jsdoc/check-param-names': 2,
+    'jsdoc/check-syntax': 2,
+    'jsdoc/check-tag-names': [2, { definedTags: ['internal', 'packageDocumentation'] }],
+    'jsdoc/check-types': 2,
+    'jsdoc/implements-on-classes': 2,
+    'jsdoc/match-description': 0,
+    'jsdoc/newline-after-description': 2,
+    'jsdoc/no-types': 0,
+    'jsdoc/no-undefined-types': 2,
+    'jsdoc/require-description': 0,
+    'jsdoc/require-description-complete-sentence': 0,
+    'jsdoc/require-example': 0,
+    'jsdoc/require-hyphen-before-param-description': 2,
```

Comment:
```suggestion
    'jsdoc/require-hyphen-before-param-description': [2, 'always', { tags: { property: 'never' } }],
```
 
There should not be a hyphen before `@property` description, otherwise the hyphen will be included in the generated description. 

something like this
```ts
    /**
     * - chunking algorithm used to build ipfs DAGs (default: `'size-262144'`)
     */
```